profanity_filter_allowed.wlist

```diff
@@ -0,0 +1,63 @@
+﻿fucc
+fuck
+fuckd
+fucked
+fucker
+fucking
+fuckr
+fuct
+fuuck
+fuuckd
+fuucked
+fuucker
+fuucking
+fuuckr
+fuuuck
+fuuuckd
+fuuucked
+fuuucker
+fuuucking
+fuuuckr
+fuuuuck
+fuuuuckd
+fuuuucked
+fuuuucker
+fuuuucking
+fuuuuckr
+fuuuuuck
+fuuuuuckd
+fuuuuucked
+fuuuuucker
+fuuuuucking
+fuuuuuckr
+fuuuuuuck
+fuuuuuuckd
+fuuuuuucked
+fuuuuuucker
+fuuuuuucking
+fuuuuuuckr
+fuuuuuuuck
+fuuuuuuuckd
+fuuuuuuucked
+fuuuuuuucker
+fuuuuuuucking
+fuuuuuuuckr
+fuuuuuuuuck
+fuuuuuuuuckd
+fuuuuuuuucked
+fuuuuuuuucker
+fuuuuuuuucking
+fuuuuuuuuckr
+fuuuuuuuuuck
+fuuuuuuuuuckd
+fuuuuuuuuucked
+fuuuuuuuuucker
+fuuuuuuuuucking
+fuuuuuuuuuckr
+fuuuuuuuuuu
+sh1t
+sh17
+Shat
+shi7
+shit
+sh|t
```

Comment:
Why are these allowed?

Reply:
It was discussed in staff channels that we didn't want to filter those words.

Reply:
Huh, but they're still swear words, surely it should be all or nothing? Also I'm not sure if they should be in the repo